packages/mdc-list/constants.ts

```diff
@@ -40,19 +40,21 @@ const strings = {
   ARIA_ORIENTATION_HORIZONTAL: 'horizontal',
   ARIA_ROLE_CHECKBOX_SELECTOR: '[role="checkbox"]',
   ARIA_SELECTED: 'aria-selected',
-  CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"]:not(:disabled), input[type="radio"]:not(:disabled)',
-  CHECKBOX_SELECTOR: 'input[type="checkbox"]:not(:disabled)',
+  CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"], input[type="radio"]',
+  CHECKBOX_SELECTOR: 'input[type="checkbox"]',
   CHILD_ELEMENTS_TO_TOGGLE_TABINDEX: `
     .${cssClasses.LIST_ITEM_CLASS} button:not(:disabled),
     .${cssClasses.LIST_ITEM_CLASS} a
   `,
+  ENABLED_CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"]:not(:disabled), input[type="radio"]:not(:disabled)',
+  ENABLED_CHECKBOX_SELECTOR: 'input[type="checkbox"]:not(:disabled)',
   FOCUSABLE_CHILD_ELEMENTS: `
     .${cssClasses.LIST_ITEM_CLASS} button:not(:disabled),
     .${cssClasses.LIST_ITEM_CLASS} a,
     .${cssClasses.LIST_ITEM_CLASS} input[type="radio"]:not(:disabled),
     .${cssClasses.LIST_ITEM_CLASS} input[type="checkbox"]:not(:disabled)
   `,
-  RADIO_SELECTOR: 'input[type="radio"]:not(:disabled)',
+  RADIO_SELECTOR: 'input[type="radio"]',
```

Comment:
This began as an effort for better naming consistency but actually fixes a bug where if the first radio element of a list is disabled, the remaining radio list breaks. This was due to the following lines in `layout()` of component not realizing a disabled radio at index 0 also makes the whole list a radio list, as a result of the constant above.  

```
 ...
 else if (this.adapter_.hasRadioAtIndex(0)) {
      this.isRadioList_ = true;
 }
```